src/main/java/nextstep/ladder/domain/LadderBoard.java

```diff
@@ -0,0 +1,23 @@
+package nextstep.ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LadderBoard {
+    private List<Line> ladderBoard = new ArrayList<>();
+
+    public LadderBoard(List<Line> ladderBoard) {
+        this.ladderBoard = ladderBoard;
+    }
+
+    public LadderBoard(int countOfHeight, int countOfPerson) {
```

Comment:
```suggestion
    public LadderBoard(int 행, int 열) {
```

`LadderBoard`의 입장에서 너무 과한 개념이 들어간 이름들 아닌가요?
`LadderBoard`내부에서는 어떻게 사용하나요?

복잡한 의미를 내포하고 있는 의미는 우리의 머릿속을 복잡하게 만들지 않을까요?
어쩌면 외부에서의 이름을 객체에게 계속 강요하고 전파하고 있는거 아닌가요?

Reply:
음.. 생각해보지 못한 내용이네요

말씀해주신대로 내부에서 사용하는 목적에 따라 네이밍을 하는 것이 좋은 것 같습니다!